### PR TITLE
Note addition

### PR DESCRIPTION
**Pull request by joinimran**
Now onwards as MFA will only work with Azure, I have added the note.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4872

### CODE REVIEW

**Comment by joinimran** [comment]
Suggestions applied.

**Comment by illfated** [comment]
@JohanFreelancer9 : Would you like to have a look at this short content, before we can declare this PR ready for final review?

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 1]
```diff
@@ -93,6 +93,9 @@ The built-in Windows Hello for Business provisioning experience creates a hardwa
 
 #### Multifactor authentication
 
+> [!IMPORTANT]
+> As of July 1, 2019, Microsoft will no longer offer MFA Server for new deployments. New customers who would like to require multi-factor authentication from their users should use cloud-based Azure Multi-Factor Authentication. Existing customers who have activated MFA Server prior to July 1 will be able to download the latest version, future updates and generate activation credentials as usual. See [Getting started with the Azure Multi-Factor Authentication Server](https://docs.microsoft.com/azure/active-directory/authentication/howto-mfaserver-deploy) for more details.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]:
```suggestion
> As of July 1, 2019, Microsoft will no longer offer MFA Server for new deployments. New customers who require multi-factor authentication for their users should use cloud-based Azure Multi-Factor Authentication. Existing customers who have activated MFA Server prior to July 1, 2019 will be able to download the latest version, future updates and generate activation credentials as usual. See [Getting started with the Azure Multi-Factor Authentication Server](https://docs.microsoft.com/azure/active-directory/authentication/howto-mfaserver-deploy) for more details.
```

File: windows/security/identity-protection/hello-for-business/hello-planning-guide.md [hunk 2]
```diff
@@ -80,8 +80,8 @@ The key trust type does not require issuing authentication certificates to end u
 
 The certificate trust type issues authentication certificates to end users.  Users authenticate using a certificate requested using a hardware-bound key created during the built-in provisioning experience.  Unlike key trust, certificate trust does not require Windows Server 2016 domain controllers (but still requires [Windows Server 2016 Active Directory schema](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-trust-prereqs#directories)).  Users can use their certificate to authenticate to any Windows Server 2008 R2, or later, domain controller.
 
->[!NOTE]
->RDP does not support authentication with Windows Hello for business key trust deployments. RDP is only supported with certificate trust deployments at this tim
+> [!NOTE]
+> RDP does not support authentication with Windows Hello for business key trust deployments. RDP is only supported with certificate trust deployments at this time.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/hello-planning-guide.md]:
```suggestion
> RDP does not support authentication with Windows Hello for Business key trust deployments. RDP is only supported with certificate trust deployments at this time.
```

**Comment by joinimran** [comment]
Good to go from my side.

**Comment by jvsam** [comment]
Hi @mapalko, this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you.

**Comment by jvsam** [comment]
Thank you for merging this PR @Dansimp. Much appreciated!

**Pull request merged**